src/test/features/response/tasks/howMuchHaveYouPaidTask.ts

```diff
@@ -0,0 +1,120 @@
+/* tslint:disable:no-unused-expression */
+import { expect } from 'chai'
+
+import { PartialAdmission, ResponseDraft } from 'response/draft/responseDraft'
+import { IndividualDetails } from 'forms/models/individualDetails'
+import { Defendant } from 'drafts/models/defendant'
+
+import { Response } from 'response/form/models/response'
+import { AlreadyPaid } from 'response/form/models/alreadyPaid'
+import { YesNoOption } from 'models/yesNoOption'
+import { HowMuchHaveYouPaid } from 'response/form/models/howMuchHaveYouPaid'
+import { ResponseType } from 'response/form/models/responseType'
+import { HowMuchHaveYouPaidTask } from 'response/tasks/howMuchHaveYouPaidTask'
+import { LocalDate } from 'forms/models/localDate'
+import { generateString } from '../../../app/forms/models/validationUtils'
+import { ValidationConstraints } from 'forms/validation/validationConstraints'
+
+const validLocalDate = LocalDate.fromObject({ day: 1, month: 1, year: 2010 })
+const validAmount = 100
+const validText = 'valid'
+
+function validResponseDraft (): ResponseDraft {
+  const responseDraft: ResponseDraft = new ResponseDraft()
+  responseDraft.response = new Response(ResponseType.PART_ADMISSION)
+  responseDraft.partialAdmission = new PartialAdmission()
+  responseDraft.partialAdmission.alreadyPaid = new AlreadyPaid(YesNoOption.YES)
+  responseDraft.partialAdmission.howMuchHaveYouPaid = new HowMuchHaveYouPaid(validAmount, validLocalDate, validText)
+  responseDraft.defendantDetails = new Defendant(new IndividualDetails())
+
+  return responseDraft
+}
+
+describe('HowMuchHaveYouPaidTask', () => {
+
+  context('should not be completed when', () => {
+
+    it('response is not partially admission', () => {
```

Comment:
this sentence doesn't make sense to me, how about:
```
response is not partial admission
```

Reply:
ok